### DISCUSSION TITLE
December weekly call thread

### DISCUSSION THREAD

**Discussion by zabil** [getgauge/taiko#1741] [Development]
This is thread to keep track of Taiko teams weekly calls and discussions during this call. The calls are open to all and will happen over zoom.

**Comment by zabil** [comment]
Dates and timings for this week's 

* 16/12/2020 10:30 - 11:30 GMT Link (https://thoughtworks.zoom.us/j/92389796277)

An agenda will be shared soon. 

If you want to be a part of this call please share

* You zoom user name
* The item you want to discuss

Please subscribe to this thread for updates

**Reply by sushil-relicx** [reply]
Hi Zabil - Unless I am mistaken, this would be very late US pacific time (2:30 or 3:30 am). Any chance, we can do it a bit more friendly US time :-)

**Reply by zabil** [reply]
Time zones are hard, but you can alternatively propose a time between 12:30 PM IST - 6 PM IST considering the time zones of the team.

**Reply by zabil** [reply]
Agenda for this call

* Taiko recorder
* Dev containers in VS Code
* Issue with running gauge tests in debug mode
* Golang plugin github actions

**Comment by sushil-relicx** [comment]
12:30 PM IST will work. Thanks for your consideration.

**Reply by zabil** [reply]
See you at 12:30 PM IST Please use the zoom link to join the call.

**Reply by sushil-relicx** [reply]
Thanks. We will be there.
